scripts/glean.py

```diff
@@ -211,11 +211,20 @@ def get_metrics(self) -> List[GleanMetric]:
                 (d[0], {**d[1], "origin": dependency["library_name"]})
                 for d in dependency_metrics.items()
             ]
+            # only get the latest version of metrics
```

Comment:
This should happen outside the for loop (it only needs to be run once, at the end) - maybe actually around line 226 would be better (just before the loop). I'd probably also give it another name, like `metric_map` or something (since it has a different structure than `metrics`, above).